misc/snapshotter/nydus-snapshotter.fscache.service

```diff
@@ -6,7 +6,7 @@ Before=containerd.service
 [Service]
 Type=simple
 Environment=HOME=/root
-ExecStart=/usr/local/bin/containerd-nydus-grpc --fs-driver fscache --daemon-mode shared --config-path /etc/nydus/config.json
+ExecStart=/usr/local/bin/containerd-nydus-grpc --fs-driver fscache --daemon-mode shared --config-path /etc/nydus/nydusd-config.fscache.json
```

Comment:
The nydusd config file is better to match what was installed by `make install`. So better use `nydusd-config.json` here?

Reply:
I think /etc/nydus/nydusd-config.json is just used for the default use.